webrender/src/render_task.rs

```diff
@@ -514,6 +524,26 @@ impl RenderTask {
         }
     }
 
+    // Check if this task wants to be made available as an input
+    // to all passes in the render task tree. To qualify for this,
+    // the task needs to have no children / dependencies.
```

Comment:
why can't we just check for `task.children.is_empty()` here then?
plus, we need to mention the A8 requirement

Reply:
Right now that's the only requirement but in the future there will be more involved logic here.